docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<p>Can you style the NB according to <a href="https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style" target="_blank">the style guide</a> please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
sure, I'm on it

Reply:
done